govcd/vm_test.go

```diff
@@ -1052,6 +1052,10 @@ func (vcd *TestVCD) Test_UpdateInternalDisk(check *C) {
 	vm, storageProfile, diskSettings, diskId, previousProvisioningValue, err := vcd.createInternalDisk(check, vmName, 1)
 	check.Assert(err, IsNil)
 
+	description := "Test_UpdateInternalDisk_Description"
```

Comment:
Please also add a comment above of why this has to be done explicitly.

Reply:
added